test/eventing.bash

```diff
@@ -39,19 +35,23 @@ function run_knative_eventing_tests {
 }
 
 function run_knative_eventing_operator_tests {
-  (
-  local exitstatus=0
   logger.info 'Running eventing operator tests'
+  (
+  local exitstatus test_namespace_saved
+  exitstatus=0
 
   checkout_knative_eventing_operator
 
-  logger.warn 'TODO(ksuszyns): Write eventing operator tests execution.'
+  test_namespace_saved="${TEST_NAMESPACE}"
```

Comment:
We're running in a subshell anyway, do we need to do this?

Reply:
Yeah, that's right. We don't need that. I'll remove it in next PR, so this can be merged, okay?